site/_docs/reference.md

```diff
@@ -1875,14 +1876,44 @@ is named as "fixed windowing".
 
 | Operator syntax      | Description
 |:-------------------- |:-----------
-| TUMBLE(table, DESCRIPTOR(column_name), interval [, time ]) | Indicates a tumbling window of *interval* for *datetime*, optionally aligned at *time*. Tumbling is applied on table in which there is a watermarked column specified by descriptor.
+| TUMBLE(table, DESCRIPTOR(datetime), interval [, time ]) | Indicates a tumbling window of *interval* for *datetime*, optionally aligned at *time*.
 
```

Comment:
Do we really support `[, time ]` ?

Reply:
yes you are right. We don't support it so far.

Do you prefer me to support it in this PR, or a future PR (I can modify documentation accordingly)?

Reply:
Either is fine, we should keep our document correct and exact.

Reply:
Done!